Update FedCM articles for Chrome 103 (#2875)
GoogleChrome/developer.chrome.com: site/en/blog/fedcm-origin-trial/index.md

description: >
  A Web Platform API that allows users to login to websites with their federated accounts in a privacy preserving manner.
date: 2022-04-25
updated: 2022-05-23
tags:
  - privacy
  - security
  - origin-trials
---

{% Aside %}

**Update, May 2022**

Starting Chrome 103, FedCM:
* Supports desktop environments.
* Supports per RP settings on desktop.

> * No longer requires privacy policy URL in [the client metadata
>   endpoint](#client-metadata-endpoint) and the client metadata endpoint itself
>   is now optional.

Reviewer: ```suggestion
* The [client metadata endpoint](#client-metadata-endpoint) is
   now optional.
   *.  In this endpoint, the privacy policy URL is also optional.
```

## Support and compatibility {: #compatibility }
  

> FedCM is supported in Google Chrome 101 and above on Android and 103 and above
> on desktop. It's not supported by other browsers yet, but the [WebKit team has
> expressed general
> support](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html)
> and interest in working together on the FedCM proposal.

Reviewer: ```suggestion
FedCM is supported in:
*  **Android**: Google Chrome 101 and above
*  **Desktop**: Google Chrome 103 and above

FedCM isn't supported by other browsers yet, but the [WebKit team has
expressed general
support](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html)
and interest in working together on the FedCM proposal.
```

### Develop and test locally with a flag {: #set-flag}

Set a [browser flag](/blog/browser-flags/) to develop locally, here is
how:

> 1.  In the Chrome address bar, type `chrome://version`. Confirm the browser
>     version is newer than 101.
> 2.  Go to `chrome://flags#fedcm` to enable FedCM.
> 3.  Restart your browser.

Reviewer: ```suggestion
1.  Open Google Chrome. In the address bar, enter `chrome://version`. Confirm
    the browser version is 101 or higher.
2.  Go to `chrome://flags#fedcm` to enable FedCM.
3.  Restart your browser.
```
